test/inductor/test_pattern_matcher.py

```diff
@@ -664,6 +665,37 @@ def fn2(inp, a, b):
         _, (code) = run_and_get_code(fn2, args[0], args[1], args[2])
         FileCheck().check_not("extern_kernels.addmm(").run(code[0])
 
+    def test_match_equivalent_function_invocations(self):
+        # pad_mm.py only registered `addmm_pattern` which explicitly passes in alpha and beta,
+        # since they have default values, we should also match the equivalent invocations like
+        # the following two cases.
+        @torch.compile()
+        def f1(inp, x, y):
+            return torch.ops.aten.addmm(inp, x, y)
+
+        @torch.compile()
+        def f2(inp, x, y):
+            return torch.ops.aten.addmm(inp, x, y, alpha=1, beta=1)
```

Comment:
These are actually the same test case, since torch_dispatch will remove default args. We might need to add custom passes to exercise normalization logic.